content/pl/port-binding.md

```diff
@@ -0,0 +1,14 @@
+## VII. Przydzielanie portów
+### Udostępniaj usługi przez przydzielanie portów
+
+Aplikacje internetowe czasem uruchamiane w ramach webserwera. Aplikacje napisane w PHP np. działają jako moduł wewnątrz [Apache HTTPD](http://httpd.apache.org/), natomiast aplikacje w Javie mogą być uruchomione wewnątrz [Tomcata](http://tomcat.apache.org/).
```

Comment:
1. Brakuje chyba `czasem SĄ` uruchamiane. A tak ogólnie, to brzmi dość potocznie - proponuję coś w stylu: "Zdarza się, że aplikacje internetowe uruchamiane są w ramach serwera web".